src/app/shared/entry-actions/workflow-actions.component.html

```diff
@@ -17,7 +17,7 @@
     id="DOIButton"
     color="accent"
     (click)="requestDOIForWorkflowVersion()"
-    [disabled]="(isRefreshing$ | async) || selectedVersion?.doiURL"
+    [disabled]="(isRefreshing$ | async) || selectedVersion?.doiURL || !zenodoAccountIsLinked"
     [matTooltip]="createdoimessage"
```

Comment:
In a perfect world, you might change the tooltip to indicate why the button is disabled (if it is), e.g., something like:  "You already have a DOI", "Please link your Zenodo credentials first", etc.

Reply:
It seems a tooltip is not displayed on a disabled button